pandas/tests/extension/test_integer.py

```diff
@@ -186,7 +186,9 @@ class TestInterface(base.BaseInterfaceTests):
 
 
 class TestConstructors(base.BaseConstructorsTests):
-    pass
+    @pytest.mark.xfail(reason="bad is-na for empty data")
```

Comment:
why is this xfailed?

Reply:
- coerce_to_array()  in core/arrays/integer.py doesn't accept array(nan).

Reply:
Ideally we would fix this here. What needs to change?

Reply:
need to allows values.ndim to be 0 in coerce_to_array().

Reply:
Can we instead not pass a 0-dim array to `coerce_to_array`? It's not clear to me why we need a 0-d array in the first place.

Reply:
I added processing to convert np.nan to [].
```
values = [] if values is np.nan else values
```